apps/vs-code-designer/src/app/commands/dataMapper/DataMapperPanel.ts

```diff
@@ -82,7 +82,7 @@ export default class DataMapperPanel {
 
   private async _setWebviewHtml() {
```

Comment:
NIT: This can be set for another PR. For now its good. For this `_setWebviewHtml` function lets remove it and reuse the one from `apps/vs-code-designer/src/app/utils/codeless/getWebViewHTML.ts`.  Its basically the same code.

Reply:
I didn't notice there was the same function, thanks! I don't see other functions that could possibly exist in the helper, but if there is, I'll change it asap.

Restructured to use the function in `getWebViewHTML.ts`, but left the `_setWebviewHtml ` since that's an async function therefore the function body cannot be moved to constructor. Changed it as a async setter function instead.